drivers/scheduler/scheduler.go

```diff
@@ -49,6 +49,8 @@ type ScheduleOptions struct {
 	Nodes []node.Node
 	// StorageProvisioner identifies what storage provider should be used
 	StorageProvisioner string
+	//DeploymentReplicas specific parameter when we need change statefulset replicas
```

Comment:
specific parameter when we need **to** change statefulset replicas

Reply:
renamed to AppReplicas since it can be used to deployments or statefulsets